credentials/apps/badges/distribution/credly/credly_badges/models.py

```diff
@@ -55,4 +55,11 @@ class CredlyBadge(UserCredential):
     """
     Earned Credly badge template for user.
     """
-    # TODO: check if we can fetch pii for username from LMS for badge issuing?
\ No newline at end of file
+
+    ISSUING_STATES = Choices("created", "no_response", "error", "pending", "accepted", "rejected", "revoked")
+
+    issuing_state = StatusField(
```

Comment:
Let's rename to `state` since it will be related for different pipelines: issuing, revocation, etc.

...and I'm expecting data migration here for the new field?